### PR TITLE
Fix errors that are caused by using deprecated functions

### PR DESCRIPTION
**Pull request by MineBill**
Just renamed some stuff to make the code compile when using the master branch of Zig.
#426

### CODE REVIEW

**Comment by treeshateorcs** [comment]
nice! thank you

**Comment by treeshateorcs** [comment]
are you 100% positive you need `ensureTotalCapacity`, and not `ensureUnusedCapacity` in these cases? can you explain why?

**Comment by MineBill** [comment]
> are you 100% positive you need `ensureTotalCapacity`, and not `ensureUnusedCapacity` in these cases? can you explain why?

If you take a look at where `ensureCapacity` was [defined](https://github.com/ziglang/zig/blob/173d56213b60fc570b6ba3922ee1d40bbf0d0e36/lib/std/array_hash_map.zig#L205) before the new deprecations, you can see it was an alias for `ensureTotalCapacity`.

**Comment by treeshateorcs** [comment]
oh, thank you!

File: src/setup.zig [hunk 1]
```diff
@@ -96,7 +96,7 @@ pub fn wizard(allocator: *std.mem.Allocator) !void {
         else => 1024 * 1024,
     };
 
-    std.debug.warn("Writing config to {s}/zls.json ... ", .{config_path});
+    std.debug.print("Writing config to {s}/zls.json ... \n", .{config_path});
```
[the remark this answers is not in the document]
Comment (MineBill) [reply src/setup.zig]: Oops, you are right.

**Comment by SuperAuguste** [comment]
Looks good, thanks!

**Pull request merged**